core/domain/platform_parameter_domain.py

```diff
@@ -23,29 +23,32 @@
 
 from constants import constants
 from core.domain import change_domain
-from core.platform import models
 import feconf
 import python_utils
 import utils
 
 
-(config_models,) = models.Registry.import_models(
-    [models.NAMES.config])
-memcache_services = models.Registry.import_memcache_services()
-
 SERVER_MODES = utils.create_enum('dev', 'test', 'prod') # pylint: disable=invalid-name
 FEATURE_STAGES = SERVER_MODES # pylint: disable=invalid-name
+DATA_TYPES = utils.create_enum('bool', 'string', 'number') # pylint: disable=invalid-name
 
 ALLOWED_USER_LOCALES = [
     lang_dict['id'] for lang_dict in constants.SUPPORTED_SITE_LANGUAGES]
-
 ALLOWED_SERVER_MODES = [
     SERVER_MODES.dev, SERVER_MODES.test, SERVER_MODES.prod]
-
 ALLOWED_FEATURE_STAGES = [
     FEATURE_STAGES.dev, FEATURE_STAGES.test, FEATURE_STAGES.prod]
+ALLOWED_CLIENT_TYPES = ['Web', 'Android']
+ALLOWED_BROWSER_TYPES = ['Chrome', 'Edge', 'Safari', 'Firefox', 'Others']
+
+# The ordering of elements in ALLOWED_APP_VERSION_FLAVOR implies the ordering
+# of corresponding flavors, which is used in app_version_flavor filter for order
+# comparison, with following ordering: 'alpha' < 'beta' < 'test' < 'release'.
+ALLOWED_APP_VERSION_FLAVOR = ['alpha', 'beta', 'test', 'release']
```

Comment:
Sorry I think I mismentioned this earlier. The relationship should be: test < alpha < beta < release (we need to test code before it reaches any users).

Reply:
Ah right -- changed the ordering.